src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
You editor settings is changing a bunch of stuff that it does not need to again.